lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -355,6 +360,79 @@ class FromDeviceOpConversionPattern
   }
 };
 
+// TypecastOp conversion pattern
+//
+class TypecastOpConversionPattern
```

Comment:
The op lives in `ttnn::experimental` namespace, you can override the `getPrefixSwapPattern()`. `GetDeviceOpConversionPattern` has an example.